lib/TransactionRunner.js

```diff
@@ -265,13 +265,13 @@ class TransactionRunner {
     // Transaction skipping (can be modified in hooks). If the input format
     // is OpenAPI 2, non-2xx transactions should be skipped by default.
     let skip = false;
-    if (transaction.apiDescriptionMediaType.includes('swagger')) {
+    if (transaction.apiDescription && transaction.apiDescription.mediaType.includes('swagger')) {
       const status = parseInt(response.status, 10);
       if ((status < 200) || (status >= 300)) {
         skip = true;
       }
     }
-    delete transaction.apiDescriptionMediaType;
+    delete transaction.apiDescription;
```

Comment:
Yes. It is currently passed along only for the purpose of checking the media type above. I have future plans with it, so I keep it as an object, but currently it is not further used anywhere else, so I'm removing it immediately after it's used, to prevent it to leak into hooks etc.